packages/python_host/src/one_sdk/app.py

```diff
@@ -80,11 +85,15 @@ def __init__(
 		sf_host_link(self._linker, self)
 
 		wasi = WasiConfig()
-		wasi.inherit_env()
 		wasi.inherit_stdout()
 		wasi.inherit_stderr()
-		self._store.set_wasi(wasi)
+
+		env = [['ONESDK_DEFAULT_USERAGENT', WasiApp.user_agent()]]
+		for var in os.environ:
+			env.append([var, os.environ[var]])
```

Comment:
nitpick: I think this can be done in a more Pythonic way, offering two variants:

This one allocates a list for the result, but I think it's easy to understand
```py
env = [*os.environ.items(), ('ONESDK_DEFAULT_USERAGENT', WasiApp.user_agent())]
```

This one is technically the best for performance because the `.env` property is a setter which only needs an iterable, not a list.
```py
import itertools
env = itertools.chain(os.environ.items(), [('ONESDK_DEFAULT_USERAGENT', WasiApp.user_agent())])
```